site/default/static/map.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
```

Comment:
What are the criteria for files to include?  Just every script file that's not a test and not in node_modules?